src/main/java/com/softserve/controller/ArchiveController.java

```diff
@@ -44,13 +44,13 @@ public ArchiveController(ArchiveService archiveService, ScheduleService schedule
     @PostMapping("/{semesterId}")
     @ApiOperation(value = "Save archive schedule by semesterId in mongo db")
     @PreAuthorize("hasRole('MANAGER')")
-    public ResponseEntity archiveScheduleBySemester(@PathVariable("semesterId") Long semesterId) {
+    public ResponseEntity<Object> archiveScheduleBySemester(@PathVariable("semesterId") Long semesterId) {
```

Comment:
why not the actual return type? ScheduleFullForArchiveDTO

Reply:
In line 52 return ResponseEntity<MessageDTO>, and 64 line return ResponseEntity<TemporaryScheduleForArchiveDTO>. And so I return ResponseEntity<Object>.